lib/engine/action/run_routes.rb

```diff
@@ -24,6 +24,7 @@ def self.h_to_args(h, game)
             revenue_str: route['revenue_str'],
             subsidy: route['subsidy'],
             halts: route['halts'],
+            ability: route['ability'],
```

Comment:
should this be multiple abilities?

Reply:
All routes will have same value for ability, but that was a simplification that I think might be ok.

I think it might be possible to have multiple route related abilities active, so maybe the code should support that? But right now there is only one so I did not implement support for that.

Reply:
Ok, I made it into a list instead, making it possible to support multiple abilities in the future.